src/modules/dashboard/components/CoinMachine/BuyTokens/BuyTokens.tsx

```diff
@@ -5,6 +5,7 @@ import * as yup from 'yup';
 import { FormikProps } from 'formik';
 import { AddressZero } from 'ethers/constants';
 import { bigNumberify } from 'ethers/utils';
+import isNil from 'lodash/isNil';
```

Comment:
This might be a nitpick, but I really hate importing crap from `lodash`

There must be a easier way just to check if a value is `null` or `undefined` without importing an extra library for that.

Why not cast it to a boolean ?

```js
 const sellableTokenDecimals = !!sellableToken ? sellableToken.decimals : 18;
```

Reply:
:thinking: That's true, didn't occur to me that I could just try casting it to boolean